01-node-tutorial/answers/03-modules.js

```diff
@@ -0,0 +1,9 @@
+const personalInformationObj = require("./04-names");
+const greetPersonFun = require("./05-utils");
+const flavorsObj = require("./06-alternative-flavor");
+
+greetPersonFun(personalInformationObj.name);
+const flavorValues = Object.values(flavorsObj);
```

Comment:
### Optional Improvement
[Iteration of Object]
While you can technically do this and clearly as you saw it does work, its usually not a great idea to iterate objects in JS. Usually, you'd just do the following:
```js
console.log(flavorsObj.flavorOne);
console.log(flavorsObj.flavorTwo);
```

The issue is in general, objects can contain anything. It could be strings like we have here or numbers or booleans or (often) functions. If you expect a string, but its actually a function things can get kinda weird to say the least. Usually though, you're code just blows up.